### ISSUE TITLE
other port logs

### ISSUE THREAD

**Issue by paulwratt** [disloops/psadify#1]
In tracking down an IP address from a series of enties in a NginX error log, I found your site. I left a reduced snippet on the RPi PSAD blog page, they were from the same address & "compaines" on your Top 10 lists. They were interesting because they showed URLS that allow backdoor entry via a web server (some of them are silly - poor security).

Can PSAD corelate more than just IPTables logs.

BTW I use RPi(s) with Alpine Linux, and RPiOS with Alpine Docker images

**Comment by disloops** [comment]
Hey @paulwratt! Thanks for your note. I see your comment here:

[https://disloops.com/psad-on-raspberry-pi/#comment-6575](https://disloops.com/psad-on-raspberry-pi/#comment-6575)

You're seeing some automated scanning for well-known vulnerabilities in your log files. In regard to your question about PSAD's capabilities, you may want to address it to the PSAD mailing list here:

[https://sourceforge.net/p/psad/mailman/](https://sourceforge.net/p/psad/mailman/)

I'd suggest adding a little more info about what you're after with PSAD to get the best feedback. I'll keep a lookout for the responses. Let me know if I can help with anything else.

**Comment by paulwratt** [comment]
Thanks

I just noticed your `psadify` page is quite comprehensive,  and wondered how much more analysis could be extracted if PSAD was also looking at web serv logs. (eg. I saw some probes to `/solr/..` which I guess might be a SolarWinds install).

I guess your just using PSAD on your firewall to get a better picture of whats trying to get in, and hence this project (`psadify`), as opposed to trying to block whats coming in (like say building a dynamic firewall blacklist) or analysing how "they" are trying to get in.

Its fine, I always like to see how others are using their RPi, especially net connected. I'll try and take some time to look at PSAD thoroughly before I post anything on the list, so thanks again (and for `psadify`).
